CMakeLists.txt

```diff
@@ -146,26 +146,70 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/src/OpenShotVersion.h
 
 ################### DOCUMENTATION ###################
 # Find Doxygen (used for documentation)
-set(DOCS_ENABLED FALSE) # Only set true if Doxygen is found and configured
-if (ENABLE_DOCS)
-  include(cmake/Modules/UseDoxygen.cmake)
+set(LIB_DOCS_ENABLED FALSE) # Only set true if Doxygen is found and configured
+if(NOT DEFINED ENABLE_LIB_DOCS)
+  set(ENABLE_LIB_DOCS ${ENABLE_DOCS})
+endif()
```

Comment:
@jonoomph Oh, right, but I do still check for the old `ENABLE_DOCS` option, just in case the user is still setting it on the command line. Purely a legacy-support thing, to avoid breaking builds (but not brains).

I should probably emit a deprecation message in that case, though, so we can eventually get rid of that variable.